spdm.rflx

```diff
@@ -0,0 +1,1496 @@
+            then Opaque_Data
+               if Opaque_Length_2 * 256 + Opaque_Length_1 <= 1024;
+         Opaque_Data : Opaque
+            with Size => 8 * (Opaque_Length_2 * 256 + Opaque_Length_1);
+         Signature : Signature;
+      end message;
+
+   -- 10.12 ERROR response message
+
+   type Error_Code is
+      (Invalid_Request        => 16#01#,
+       Invalid_Session        => 16#02#,
+       Busy                   => 16#03#,
+       Unexpected_Request     => 16#04#,
+       Unspecified            => 16#05#,
+       Decrypt_Error          => 16#06#,
+       Unsupported_Request    => 16#07#,
+       Request_In_Flight      => 16#08#,
+       Invalid_Response_Code  => 16#09#,
+       Session_Limit_Exceeded => 16#0A#,
+       Major_Version_Mismatch => 16#41#,
+       Response_Not_Ready     => 16#42#,
+       Request_Resynch        => 16#43#,
+       Vendor                 => 16#FF#)
+   with Size => 8;
+
+   type Error_Session_ID is mod 2**8;
+
+   type RDT_Exponent is mod 2**8;
+   type Token is mod 2**8;
+   type RDTM is range 1 .. 2**8 - 1 with Size => 8;
+
+   type Response_Not_Ready_Data is
+      message
+         RDT_Exponent : RDT_Exponent;
+         Request_Code : Request_Code;
+         Token        : Token;
+         RDTM         : RDTM;
+      end message;
+
+   type Zero_Error_Data is range 0 .. 0 with Size => 8;
+
+   type Vendor_ID_Len is mod 2**8;
+   type Vendor_ID_Double is mod 2**16;
+   type Vendor_ID_Quad is mod 2**32;
+
+   --  ISSUE: Componolit/RecordFlux-specifications#92
+   --  This message differs between SPDM version 1.0 and 1.1.
+   --  Since the minor version is not accessible from
+   --  within this message one message is needed for each version
+   --  that is selected by an outer message.
+   type Error_Response_1_0 is
+      message
+         Error_Code : Error_Code
+            then Zero_Error_Data
+               if Error_Code = Invalid_Request
+                  or Error_Code = Busy
+                  or Error_Code = Unexpected_Request
+                  or Error_Code = Unspecified
+                  or Error_Code = Major_Version_Mismatch
+                  or Error_Code = Response_Not_Ready
+                  or Error_Code = Request_Resynch
+            then Request_Response_Code
+               if Error_Code = Unsupported_Request
+            then Registry_ID
+               if Error_Code = Vendor;
+         Zero_Error_Data : Zero_Error_Data
+            then Response_Not_Ready_Data
+               if Error_Code = Response_Not_Ready
+            then null
+               if Error_Code /= Response_Not_Ready;
+         Request_Response_Code : Request_Code
+            then null;
+         Response_Not_Ready_Data : Response_Not_Ready_Data
+            then null;
+         Registry_ID : Registry_ID;
+         Vendor_ID_Len : Vendor_ID_Len
+            then Opaque_Error_Data
+               with Size => Message'Size - (Vendor_ID_Len'Last - Error_Code'First + 1)
+                  if Vendor_ID_Len = 0
```

Comment:
Wrong indentation.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/4b2412a417abf4c2f782dc48e50069dea280daf2.